kernel/aster-nix/src/fs/ext2/inode.rs

```diff
@@ -886,6 +887,49 @@ impl InodeImpl_ {
         }
     }
 
+    pub fn read_blocks_sync(
+        &self,
+        start_bid: Ext2Bid,
+        nblocks: usize,
+        raw_buf: &mut [u8],
```

Comment:
My concern is that the `read_blocks_sync` uses slice as the input, but the  `read_block_sync` uses `&Frame` as the input, can we unify the APIs with `VmReader` or `VmWriter`?

Reply:
Worth give it a try

Reply:
Now I introduce a unified function of inode which is capable to read/write both one or multiple block(s). Please review it again, thanks!